### PR TITLE
Document the availability of the os.CLONE_ constants

### PR DESCRIPTION
**Pull request by srittau**
Consolidate allowlists

### CODE REVIEW

**Review by AlexWaygood — APPROVED** [review]

File: stdlib/@tests/stubtest_allowlists/linux.txt [hunk 1]
```diff
@@ -35,6 +34,10 @@ multiprocessing.popen_spawn_win32
 fcntl.I_[A-Z0-9_]+
 os.SCHED_[A-Z_]+
 posix.SCHED_[A-Z_]+
+(posix.O_[A-Z_]+)?
+(posix.ST_[A-Z]+)?
+(termios.[A-Z0-9_]+)?
+((os|posix).CLONE_[A-Z]+)?
```
Comment (srittau) [inline stdlib/@tests/stubtest_allowlists/linux.txt:40]: While the `CLONE_` constants were only added in Python 3.12, I think having them in the common file is better for maintainability.

Comment (AlexWaygood) [reply stdlib/@tests/stubtest_allowlists/linux.txt:40]: Maybe add a comment to the specific allowlist entry noting they were added on py312?

File: stdlib/@tests/stubtest_allowlists/linux.txt [hunk 2]
```diff
@@ -35,6 +34,10 @@ multiprocessing.popen_spawn_win32
 fcntl.I_[A-Z0-9_]+
 os.SCHED_[A-Z_]+
 posix.SCHED_[A-Z_]+
+((os|posix).CLONE_[A-Z]+)?  # Python 3.12+
```
Comment (srittau) [inline stdlib/@tests/stubtest_allowlists/linux.txt:37]: While the CLONE_ constants were only added in Python 3.12, I think having them in the common file is better for maintainability.

File: stdlib/@tests/stubtest_allowlists/linux.txt (hunk 1 above)
Comment (srittau) [reply stdlib/@tests/stubtest_allowlists/linux.txt]: I just did that while reordering them. 🎉

**Review by AlexWaygood — APPROVED** [review]

**Pull request merged**